multi-arch-builders/provisioning/ppc64le/README.md

```diff
@@ -0,0 +1,57 @@
+# OpenTofu
+
+ OpenTofu is a Terraform fork, is an open-source infrastructure as code (IaC) tool
+ lets you define both cloud and on-prem resources in human-readable configuration files
+ that you can version, reuse, and share.
+
+ To proceed with the next steps, ensure that 'tofu' is installed on your system.
+ See: https://github.com/opentofu/opentofu/releases
+
+## Before starting
+
+### PowerVS credentials
+
+ - Ensure that you have access to our account.
+ - Verify that the Fedora CoreOS image has been uploaded to the designated bucket.
+   - See documetation in how to upload the image:
+       https://cloud.ibm.com/docs/power-iaas?topic=power-iaas-deploy-custom-image
+### PowerVs Issues
+
+ - PowerVS seems to encounter a problem in creating the default local IP with the default route,
+resulting in issues to ssh to the server post-boot.
+To mitigate this, we've incorporated networking configurations into the Ignition file. However,
+we still with one issue during the Splunk Butane configuration, where the CA certification couldn't be
+downloaded during provisioning. If you encounter this issue, comment out the Red Hat CA download step
+and perform it manually on the machine after provisioning. 
+
+ - Additionally, it's important to note that PowerVS lacks the user data field in the interface for providing
```

Comment:
For the interface, is just to keep track of this issue to.

Reply:
but I don't understand. I'm specifically talking about

> Additionally, it's important to note that PowerVS lacks the user data field in the interface for providing

but if I look at the main.tf we are using:

```
    pi_user_data          = base64encode(data.ct_config.butane.rendered)
```

so that seems like we are passing user data, but the comment says PowerVS doesn't support user data?  Maybe I'm misinterpreting the comment?

Reply:
A, I probably should say in web interface, instead of `user data field in the interface `